packages/odie-client/src/components/message/feedback-form.tsx

```diff
@@ -115,9 +112,7 @@ export const FeedbackForm = ( { chatFeedbackOptions }: FeedbackFormProps ) => {
 					{ ! isFormHidden && (
 						<div ref={ feedbackRef } className="odie-conversation-feedback__message">
 							<h3>{ __( 'Thank you for your input' ) }</h3>
-							<p>
-								{ __( 'Please share any other details that can help understand your rating.' ) }
-							</p>
+							<p>{ __( 'Please share any details that can help us understand your rating' ) }</p>
```

Comment:
These need text domains. This may explain https://linear.app/a8c/issue/DOTCOM-12648/help-center-untranslated-welcome-message